client/landing/jetpack-cloud/sections/pricing/jpcom-footer/index.tsx

```diff
@@ -0,0 +1,249 @@
+						<li className="jpcom-footer__link">
+							<ExternalLink href="https://developer.jetpack.com/">
+								{ translate( 'Developers Site' ) }
+							</ExternalLink>
+						</li>
+						<li className="jpcom-footer__link">
+							<ExternalLink href="https://jetpack.com/beta/">
+								{ translate( 'Beta Program' ) }
+							</ExternalLink>
+						</li>
+						<li className="jpcom-footer__link">
+							<ExternalLink href="https://jetpack.com/contribute/">
+								{ translate( 'Contribute to Jetpack' ) }
+							</ExternalLink>
+						</li>
+						<li className="jpcom-footer__link">
+							<ExternalLink href="https://jetpack.pro/">
+								{ translate( 'Developer Network' ) }
+							</ExternalLink>
+						</li>
+						<li className="jpcom-footer__link">
+							<ExternalLink href="https://automattic.com/work-with-us/?utm_source=jetpackcom&utm_medium=link">
+								{ translate( 'Work With Us' ) }
+							</ExternalLink>
+						</li>
+					</ul>
+				</div>
+				<div className="jpcom-footer__col">
+					<h4 className="jpcom-footer__col-header">{ translate( 'More from Jetpack' ) }</h4>
+
+					<ul className="jpcom-footer__links">
+						<li className="jpcom-footer__link">
+							<ExternalLink href="https://jetpackcrm.com/">
+								{ translate( 'Jetpack CRM' ) }
+							</ExternalLink>
+						</li>
+					</ul>
+					<div className="jpcom-footer__sub-col">
+						<h4 className="jpcom-footer__col-header">{ translate( 'Partnership' ) }</h4>
+						<ul className="jpcom-footer__links">
+							<li className="jpcom-footer__link">
+								<ExternalLink href="https://jetpack.com/for/affiliates/">
+									{ translate( 'Become an Affiliate' ) }
+								</ExternalLink>
+							</li>
+							<li className="jpcom-footer__link">
+								<ExternalLink href="https://jetpack.com/for/agencies/">
+									{ translate( 'Become an Agency' ) }
+								</ExternalLink>
+							</li>
+							<li className="jpcom-footer__link">
+								<ExternalLink href="https://jetpack.com/for/hosts/">
+									{ translate( 'Become a Partner' ) }
+								</ExternalLink>
+							</li>
+							<li className="jpcom-footer__link">
+								<ExternalLink href="https://jetpack.com/hosting/#criteria">
+									{ translate( 'Listing Criteria' ) }
+								</ExternalLink>
+							</li>
+						</ul>
+					</div>
+				</div>
+			</footer>
+			<hr className="jpcom-footer__separator" />
+			<div class="jpcom-footer__plugs">
+				<div class="jpcom-footer__plugs-group">
+					<div class="jpcom-footer__a8c-services-plug">
+						<SocialLogo icon="wordpress" size={ 20 } />
+						<ExternalLink href="https://wordpress.com/" title="Powering WordPress.com">
+							{ translate( 'Powering WordPress.com' ) }
+						</ExternalLink>
+					</div>
+
+					<div class="jpcom-footer__a8c-attr-plug">
+						<ExternalLink
+							href="http://automattic.com"
+							title={ translate( 'Automattic – makers of WordPress.com and more!' ) }
+						>
+							<svg
```

Comment:
Perhaps this could be extracted in a component for readability. I doubt it'll be used somewhere else, though.

Reply:
Yes. I doubt it will be used somewhere else, so I extracted into its own component and left it in the same file. Thanks for the idea!